### PR TITLE
Add upgradable packages list component

### PR DESCRIPTION
**Pull request by jamie-suse**
# Description

This PR add an upgradable packages list component.

Example output:

image

## How was this tested?

Added unit test and Storybook demo.

### CODE REVIEW

File: assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      <h1 className="text-3xl w-4/5 p-4">
+        <span className="font-medium">Upgradable Packages: </span>
+        <span
+          className={classNames(
+            'font-bold truncate w-60 inline-block align-top'
+          )}
+        >
+          {hostname}
+        </span>
+      </h1>
```
Comment (dottorblaster) [inline assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx:63]: I think we can move all of this into the page component :+1:

File: assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                {advisory_name}
+              </div>
+            ))}
+          </div>
+        ),
+      },
+    ],
+  };
+
+  const data = upgradablePackages.map((up) => {
```
Comment (dottorblaster) [inline assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx:41]:
```suggestion
  const data = upgradablePackages.map((package) => {
```

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a couple things then we can merge, nice job! :muscle:

File: assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx (hunk 2 above)
Comment (jamie-suse) [reply assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx:41]:
```
Parsing error: The keyword 'package' is reserved

Identifier expected. 'package' is a reserved word in strict mode. Modules are automatically in strict mode.
```

Comment (dottorblaster) [reply assets/js/common/UpgradablePackagesList/UpgradablePackagesList.jsx]: Frown

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**